lib/activerecord-import/import.rb

```diff
@@ -598,6 +598,7 @@ def set_attributes_and_mark_clean(models, import_result, timestamps)
       return if models.nil?
       models -= import_result.failed_instances
       import_result.ids.each_with_index do |id, index|
+        id.tr!('()', '') if id.is_a? String
```

Comment:
Hum, now the format of result.ids are different whether you import as models or arrays (because set_attributes_and_mark_clean is only called for models and because `tr!` modifies the object).

Also it would cause problems if someone is using string as primary key and they happen to contain `(` or `)`. Unlikely, but possible.

Could we parse ids in database adapter, and return an array of arrays, like `[[1, 2], [3, 4]]`?